html_css/v2/foundations/css-foundations.md

```diff
@@ -0,0 +1,425 @@
+### Introduction
+
+There are a lot of CSS properties and concepts we can use to make some stunning websites and apps, so many that you could code for a year and still discover new properties you've never heard of before. In this lesson we're only going to focus on what we believe are some foundational concepts, things that anyone should know from the beginning, whether they are just starting out or simply need a refresher.
+
+### Learning Outcomes
+
+By the end of this lesson, you should be able to:
+
+* Add CSS to an HTML file
+* Add a class and an ID to an HTML element
+* Use element, class, and ID selectors
+* Group multiple selectors and use combinators
+* Understand what the cascade is
+* Understand how specificity, inheritance, and rule order work
+
+### Basic Syntax
+
+At the most basic level, CSS is made up of various rules. These rules are made up of a selector (more on these in a bit) and a semi-colon separated list of declarations, with each of those declarations being made up of a property:value pair. The actual syntax will be slightly different depending on which method is used for adding CSS, but generally each method will all follow this structure. 
+
+So, what exactly are these methods?
+
+#### Inline CSS
```

Comment:
two things here:
1. I think I might prefer going straight to `selectors` and saving "how do you get your CSS into your HTML" until the end.  I'm unsure about this though so take it or leave it.

2. Put the Inline CSS option last. I don't like starting the first lesson straight away with something we're going to tell them not to use LOL.

I'd word it something like this:
> #### Inline CSS
> it _is_ technically possible to stick styles directly in your html, but this approach is not recommended..... (followed by your actual content)

Reply:
1. I get what you mean. I'll mull that over because it can also make sense to order things in that way. It's kind of a "you should know how to add CSS to your HTML before learning what you can do with CSS" vs "how can you add CSS to your HTML when you don't know what you can do with it" sort of thing.

2. Would you think the order should just be reversed completely? External -> Internal -> Inline, so it's more "Here's the most common method. Here's another way you could do it, and also a third way but it's not recommended."